src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -132,8 +133,19 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
                     otlpLogRecord.Flags = (uint)logRecord.TraceFlags;
                 }
 
-                // TODO: Add additional attributes from scope and state
-                // Might make sense to take an approach similar to https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/897b734aa5ea9992538f04f6ea6871fe211fa903/src/OpenTelemetry.Contrib.Preview/Internal/DefaultLogStateConverter.cs
+                int scopeDepth = -1;
+                logRecord.ForEachScope(ProcessScope, otlpLogRecord);
+
+                void ProcessScope(LogRecordScope scope, OtlpLogs.LogRecord otlpLog)
+                {
+                    scopeDepth++;
+                    foreach (var scopeItem in scope)
+                    {
+                        var scopeItemWithDepthInfo = new KeyValuePair<string, object>($"[Scope.{scopeDepth}]:{scopeItem.Key}", scopeItem.Value);
```

Comment:
I anticipate users wanting to change this prefix behavior, but something we can tackle later 😄

Reply:
Might be nice to avoid the new string allocation each time, doesn't block this PR though.

Reply:
Yes. The description has this left as open question.
Merging to make progress. (OTLP Log Exporter is now almost ready feature-wise, except the SDK level issue related to buffering : https://github.com/open-telemetry/opentelemetry-dotnet/issues/2905)